camel/toolkits/hybird_browser_toolkit/browser_session.py

```
@@ -89,29 +89,43 @@ def __init__(
+        loop.
+        """
         # Check if we need to reuse or create a session for this event loop
         try:
             loop = asyncio.get_running_loop()
         except RuntimeError:
             raise RuntimeError(
-                "ensure_browser() "
-                "must be called "
-                "from within an async context"
+                "ensure_browser() must be called from within an async context"
             )
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)